### PR TITLE
popup-display with disabled background

### PR DESCRIPTION
**Pull request by sarahkami**
1. The Popup has now the right wording according to the mockup. We made a function to read out the urgent module-titles, and pass it to the popup.
2. Also the area of the Popup is increased to the whole page, so nothing in the background is clickable. 
@bastianalbers and @lauralindal, feel free to review ;)

### CODE REVIEW

File: src/PopupAlert.js [hunk 1]
```diff
@@ -3,10 +3,19 @@ import React from 'react';
 class Popup extends React.Component {
   render() {
     return (
-      <div className="popup">
-        <p> Du hast dringende Kurse, möchtest du diese auswählen? </p>
-        <button onClick={this.props.selectUrgentModules}> Alle auswählen </button>
-        <button onClick={this.props.dismissPopup}> Dismiss </button>
+      <div>
```
Comment (bastianalbers) [inline src/PopupAlert.js:6]: The outer div here can already have the className

File: public/css/custom-style.css [hunk 2]
```diff
@@ -255,13 +255,37 @@ footer{
...
   border: 3px solid #FF5F00;
+  border-radius:10px;
   left: 25%;
-  margin: auto;
+  margin:0 auto;
   padding: 10px;
-  position: fixed;
-  top: 50%;
-  width: 50%;
+  position: absolute;
```
Comment (bastianalbers) [inline public/css/custom-style.css:276]: make this one fixed as well, with top: 25% and height: 50vh

File: public/css/custom-style.css [hunk 3]
```diff
@@ -255,13 +255,37 @@ footer{
 
 /* ------- Popup ------------- */
 
+.cover {
+  background:rgba(0,0,0,0.2);
+  height:100%;
```
Comment (bastianalbers) [inline public/css/custom-style.css:260]: make this height: 100vh; for 100% of viewport

**Review by bastianalbers — APPROVED** [review]
Definitely _looks_ good now :+1:

**Comment by lauralindal** [comment]
@sarahkami would you like to add the change suggestions?

**Comment by bastianalbers** [comment]
Well, i made the comments before @sarahkami made her latest changes which also work, so i approved of this.

**Comment by lauralindal** [comment]
@bastianalbers ah, apologies, my mistake @sarahkami 
so i shall merge! :)

**Pull request merged**